problemsets-mat223/modules/module3.tex

```diff
@@ -105,7 +105,28 @@
 \begin{example}
 	Let $\vec u=\mat{-1\\2}$ and $\vec v=\mat{1\\-2}$. Find $\Span\Set{\vec u,\vec v}$.
 
-	XXX Finish
+	By the definition of span,
+	\[
+	\Span\Set{\vec u,\vec v}=
+	\Set{\vec x\given \vec x=\alpha\vec u+\beta\vec v\text{ for some scalars }\alpha,\beta}.
+	\]
+	Solving for when $\mat{x\\y} = \alpha\mat{-1\\2}+\beta\mat{1\\-2}$ is consistent will
```

Comment:
-> We need to determine for which $x$ and $y$ the vector equation ... is consistent.